### DISCUSSION TITLE
Performance struggle. [iOS]

### DISCUSSION THREAD

**Discussion by Latschil** [ppy/osu#22469] [Q&A] [answered]
Hello everyone,

i noticed that I don't have stable FPS ingame on Mobile.
I tried all variations of the graphic options but nothing works good. I'm getting so many stutter and drops that it's not really fun to play. It's kinda smooth but not really.
Best performance i get is on Basically Unlimited and Single Thread. It jumps between 36 and 60fps all the time. Another question is, is that 60fps capped because Display Refresh Rate? I mean at Multithread i have 120fps when im Focus the App Switcher and not the game itself.
I'm running on iPhone 12 Pro (OS 16.2).

greetings.

**Comment by peppy** [comment, marked as answer]
Performance and optimisation considerations are an ongoing effort. Please be patient and wait for future optimisations, or upgrade your device.

Also please do NOT change the frame limiter on mobile. It will not help.
